src/Bootstrap/LoadConfiguration.php

```diff
@@ -22,43 +23,8 @@ public function bootstrap(Application $app)
 
         $app->instance('config', $config = new Repository($items));
 
-        $this->loadConfigurationFiles($app, $config);
+        $this->loadConfigurationFiles($app, __DIR__.'/../../fixture/config/');
 
         mb_internal_encoding('UTF-8');
     }
-
-    /**
-     * Load the configuration items from all of the files.
-     *
-     * @param  \Illuminate\Contracts\Foundation\Application  $app
-     * @param  \Illuminate\Contracts\Config\Repository  $config
-     *
-     * @return void
-     */
-    protected function loadConfigurationFiles(Application $app, RepositoryContract $config)
```

Comment:
I definitely don't want to introduce a property, and you can already set different config path (replacing the default fixture) early on without this PR.